src/chain/member_info.rs

```diff
@@ -82,6 +84,10 @@ impl MemberInfo {
         self.age_counter.increment()
     }
 
+    pub fn increment_age(&mut self) {
+        self.age_counter = AgeCounter::from_age(self.age_counter.age().saturating_add(1))
```

Comment:
why not just call `self.increment_age_counter` or `self.age_counter.increment()` ?

Reply:
because we need to increment age, not age counter.

Reply:
ok. I got what you mean.
So basical we want the new age_counter to be reset with a new increased age, right?

Reply:
Yes. So if the age counter is e.g. 16 (corresponding to age 4), we want to go to 32, not 17.